tests/integration/solidity_contracts/PlainOpcodes/test_plain_opcodes.py

```diff
@@ -188,6 +188,37 @@ async def test_should_create_counters(
             ).result.nonce
             assert nonce_final == nonce_initial + count
 
+        @pytest.mark.skip(
+            """
+            TODO: need to fix when there is no return data from the bytecode execution,
+            it calls CallHelper instead of CreateHelper when finalizing calling context
+            """
+        )
+        @pytest.mark.parametrize("bytecode", ["0x", "0x6000600155600160015500"])
+        async def test_should_create_contract_with_bytecode_and_no_return_data(
```

Comment:
the sentence is a bit misleading to me because a contract itself doesn't have `return_data`

```
test_should_create_empty_contract_when_creation_code_has_no_return
```

?